models/repo/repo_list.go

```diff
@@ -745,3 +745,31 @@ func GetUserRepositories(ctx context.Context, opts *SearchRepoOptions) (Reposito
 	repos := make(RepositoryList, 0, opts.PageSize)
 	return repos, count, db.SetSessionPagination(sess, opts).Find(&repos)
 }
+
+func GetPrimaryRepoLanguageList(ctx context.Context, ownerID int64, user *user_model.User) (LanguageStatList, error) {
+	languageList := make(LanguageStatList, 0)
+
+	q := db.GetEngine(ctx).
```

Comment:
```suggestion
	sess := db.GetEngine(ctx).
```
We usually call this variable `sess`. Keep the tradition.